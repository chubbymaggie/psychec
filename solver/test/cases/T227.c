void f()
{
    a = (struct A*) (intptr_t)(!NULL);
}
