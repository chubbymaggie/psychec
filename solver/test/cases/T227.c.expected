/* Forward declarations */

/* Type definitions */
struct A {int dummy; } ;

/* Variables and functions */
 struct A* a ;
